### ISSUE TITLE
[Suggestion] Add a setting to adjust or shadows on grass?

### ISSUE THREAD

**Issue by korvl38** [117HD/RLHD#415]
Hi so in areas with long grass like piscatoris the shadows are very strong, with every blade of grass getting shadows its a bit too much contrast for my eyes. I know I could just turn it off but my preference is for HD. 

Actually even when shadows are turned off, each grass is still rendered. At some angles it just looks like 
 a million dark spikes all over the floor. Could we please have a small setting to adjust this to our liking? Thanks

**Comment by Jin-Jiyunsun** [comment]
Could you provide a screenshot of your problem? We already disable most grass from casting shadow.

**Comment by korvl38** [comment]
> Could you provide a screenshot of your problem? We already disable most grass from casting shadow.

Yes of course. I have turned my shadows off, see the comparison. I think its the shadow on the grass itself, everything looks very "spikey" I hope that makes sense. 

I know probably most people don't care but it's very tiring on my eyes. You could have a few levels of detail between SD and the current grass. 
![2024-04-15_10-32-47](https://github.com/117HD/RLHD/assets/101873426/a4665076-4a50-4824-a68d-f1f2f9dd4445)
![RuneLite_0ntgK1QoLb](https://github.com/117HD/RLHD/assets/101873426/569ca31b-85f3-461e-ad24-735a3a49765d)
![2024-04-15_10-32-35](https://github.com/117HD/RLHD/assets/101873426/f00f91d7-4799-48b9-a91a-f89a210b6b10)

**Comment by Jin-Jiyunsun** [comment]
I will take a look into a soultion for this but I cant make any guarantees

**Comment by Jin-Jiyunsun** [comment]
I'll drop a WIP, thoughts?

![image](https://github.com/117HD/RLHD/assets/37377657/90cd502d-fdfa-4089-8ee3-bdb79737194a)

**Comment by korvl38** [comment]
@Jin-Jiyunsun thanks for the quick WIP. It's looking much better already. Seems you have AA on when I did not but either way it looks more natural and less strong on each individual grass. 

I prefer closer to low resolution for grass but everyone has different taste, what kind of options are you able to add for adjusting grass settings between SD or HD?

**Comment by Jin-Jiyunsun** [comment]
I'll be attempting to tune it so it just looks good on a single setting, not really within the scope to offer multiple options for things like this.
